network-mapper/templates/mapper-deployment.yaml

```diff
@@ -25,6 +25,16 @@ spec:
           env:
             - name: OTTERIZE_DEBUG
               value: {{ .Values.debug | quote }}
+            - name: OTTERIZE_CLOUD_API_ADDRESS
```

Comment:
This variable should be named OTTERIZE_API_ADDRESS, so it's the same name as the CLI configuration. Need to change it in the network mapper itself too. It should look like this: https://github.com/otterize/intents-operator/blob/develop/src/shared/otterizecloud/otterizecloudclient/cloud_client.go#L16

Reply:
Did it https://github.com/otterize/network-mapper/pull/52